source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -181,36 +190,35 @@ Create a symlink for the network overlays:
 
 .. note::
 
-   |CL| installs CNI plugins that are part of the `cloud-native-basic` bundle to :file:`/usr/libexec/cni`. The directory is required because `swupd verify` uses it, if necessary, to repair a system to a known good state.
-
-.. TODO: Verify whether to omit this section per mythi, eadamsintel.
+   |CL| installs CNI plugins that are part of the `cloud-native-basic`   bundle to :file:`/usr/libexec/cni`. The directory is required because `
+   swupd verify` uses it, if necessary, to repair a system to a known good
+   state.
 
-.. **Notes about Weave Net add-on**
+**Notes about Weave Net add-on**
 
-.. If you choose the `Weave Net` add-on, then you must make the following
-.. changes because it installs itself in the :file:`/opt/cni/bin` directory.
+If you choose the `Weave Net` add-on, then you must make the following
+changes because it installs itself in the :file:`/opt/cni/bin` directory.
 
-.. For using CRI-O and `Weave Net`, you must complete the following
-.. steps.
+For using CRI-O and ``Weave Net``, you must complete the following
+steps.
 
-.. #.  Edit the :file:`/etc/crio/crio.conf` file to change `plugin_dir` from:
+#. Edit the :file:`/etc/crio/crio.conf` file to change `plugin_dir` from:
 
-..     ..  code-block:: bash
+   ..  code-block:: bash
 
-..         plugin_dir = "/usr/libexec/cni/"
+       plugin_dir = "/usr/libexec/cni/"
 
-..     to:
+    to:
 
-..     ..  code-block:: bash
+    ..  code-block:: bash
 
-..         plugin_dir = "/opt/cni/bin"
+        plugin_dir = "/opt/bin/cni"
```

Comment:
I ran `swupd update` to 26800, and afterward, the new directory shows plugins reside here: 
`~/opt/bin/cni/cni/`
Shouldn't this line be revised to reflect that?

Reply:
the from...to is unnecessary and can be dropped since the default is correct for weave. Only the symlink to loopback is needed for weave